pytest_splunk_addon/standard_lib/fields_tests/test_templates.py

```diff
@@ -149,9 +150,16 @@ def test_props_fields_no_dash_not_empty(
             record_property("results", results.as_list)
             pp = pprint.PrettyPrinter(indent=4)
             result_str = pp.pformat(results.as_list[:10])
+
+            query_for_unique_events = base_search + " | dedup punct | head 5"
```

Comment:
Use constant naming convention for the query, it's not going to be changed.

Reply:
code refactored 
added two constants:
TOP_FIVE_STRUCTURALLY_UNIQUE_EVENTS_QUERY_PART = " | dedup punct | head 5"
COUNT_BY_SOURCE_TYPE_SEARCH_QUERY_PART = " | stats count by sourcetype"